code/game/objects/items/melee/hf.dm

```diff
@@ -0,0 +1,152 @@
+/obj/item/melee/hfblade
+	name = "high frequency blade"
+	desc = "RULES OF NATURE"
+	icon_state = "hfblade"
+	worn_icon_state = "hfblade"
+	icon = 'icons/obj/hfblade.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	flags_1 = CONDUCT_1
+	obj_flags = UNIQUE_RENAME
+	force = 0.001
+	throwforce = 20
+	w_class = WEIGHT_CLASS_BULKY
+	block_chance = 50
+	armour_penetration = 75
+	sharpness = SHARP_EDGED
+	attack_verb_simple = list("chopped", "sliced", "cut", "zandatsu'd")
+	attack_verb_continuous = list("chopped", "sliced", "cut", "zandatsu'd")
+	hitsound = 'sound/weapons/rapierhit.ogg'
+	custom_materials = list(/datum/material/iron = 10000)
+	light_color = "#40ceff"
+	var/rules_of_nature = TRUE // Turn this off to break the rules, and watch the horrors of no proximity flags unfold.
+	var/brazil = FALSE
+	var/brightness = 5
+
+/obj/item/melee/hfblade/Initialize()
+	. = ..()
+	AddComponent(/datum/component/butchering, 30, 95, 5)
+	set_light(brightness)
+	START_PROCESSING(SSobj, src)
+
+/obj/item/melee/hfblade/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/item/melee/hfblade/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(attack_type == PROJECTILE_ATTACK)
+		final_block_chance = 0
+	if(attack_type == MELEE_ATTACK) // It really is not a good idea to get close to this thing.
+		final_block_chance = 100
```

Comment:
Not sure about this. Id accept 75% but 100 is a bit much

Reply:
sleeping carp gives you 100%